code/datums/candidate_poll.dm

```diff
@@ -131,3 +132,12 @@
 
 /datum/candidate_poll/proc/time_left()
 	return duration - (world.time - time_started)
+
+/datum/candidate_poll/proc/announce_chosen(list/poll_recipients)
+	if(!length(chosen_candidates))
+		return
+	for(var/mob/poll_recipient as anything in poll_recipients)
+		for(var/mob/chosen as anything in chosen_candidates)
+			if(!chosen)
```

Comment:
What is this check meant to do? If it's to check for null you should do ``isnull`` instead, if it's to check if chosen is a mob then you should just get rid of ``as anything``